src/EPR.Payment.Facade/wwwroot/index.html

```diff
@@ -227,7 +227,7 @@
             console.log('Request Data:', requestData);
 
             try {
-                const response = await fetch('/api/v1/payments', {
+                const response = await fetch('/api/v1/onlinepayments', {
```

Comment:
Same, should be online-payments

Reply:
in onlinePayment.js it is =>  const response = await fetch('/api/v1/online-payments',